radar/2022-12-05/terraform.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Terraform"
+ring: trial
+quadrant: tools
+featured: true
+---
```

Comment:
For this two, I think also assess who is the main user during the software cycle
Pulumi is more dev oriented and Terraform not